aiohttp/resolver.py

```diff
@@ -85,19 +85,49 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self._resolver = aiodns.DNSResolver(*args, loop=self._loop, **kwargs)
 
     async def resolve(
-        self, host: str, port: int = 0, family: int = socket.AF_INET
+        self, hostname: str, port: int = 0, family: int = socket.AF_INET
```

Comment:
The parent class names this `host`, so should probably keep that name (not sure why mypy doesn't give an error about this...).

Reply:
I was matching it to ThreadedResolver. I guess we change both to host?

Reply:
Probably makes more sense.